lists/vn/vn-most.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": {
+    "en": "Ministry of Science and Technology",
+    "local": " Bộ Khoa học và Công nghệ"
+  },
+  "url": "https://www.most.gov.vn/en/Pages/home.aspx",
+  "description": {
+    "en": "The Ministry of Science and Technology (MOST, Vietnamese: Bộ Khoa học và Công nghệ) is a government ministry in Vietnam responsible for state administration of science and technology activities; development of science and technology potentials; intellectual property; standards, metrology and quality control; atomic energy, radiation and nuclear safety"
+  },
+  "coverage": [
+    "VN"
+  ],
+  "code": "VN-MOST",
+  "listType": "primary",
+  "access": {
+    "availableOnline": false,
+    "onlineAccessDetails": null,
+    "publicDatabase": "http://",
+    "guidanceOnLocatingIds": "The Registration Office of Science and Technology activities (hereinafter called the Registration Office) is a professional agency, which assists the Minister to fulfill functions of management and registration of scientific research organizations with the Ministry’s authority. Organisations are provided with registration certificate. There is no public database of the registration numbers, so organisations need to provide their registration certificate. The number is located in the registration certificate and starts with a letter, followed by a dash and a three digit number.",
+    "exampleIdentifiers": "VN-MOST-A-399",
```

Comment:
You can leave out the VN-MOST. And normalising the id by removing the '-' is [recommended](https://org-idguide-handbook.readthedocs.io/en/latest/schema/#) (and should be recommended in the "guidanceOnLocatingIds"). So here, the example would be "A399".

Reply:
Edited in latest commit.